src/html/HTMLWriter.jl

```diff
@@ -2194,11 +2234,11 @@ function domify(dctx::DCtx, node::Node, i::ImageElements)
     @tags video img a
 
     if occursin(r"\.(webm|mp4|ogg|ogm|ogv|avi)$", url)
-        video[:src => url, :controls => "true", :title => alt](
+        return video[:src => url, :controls => "true", :title => alt](
             a[:href => url](alt)
         )
     else
-        img[:src => url, :alt => alt]
+        return img[:src => url, :alt => alt]
     end
```

Comment:
Is this by the formatter? I would have expected a `return` in front of the `if`.

Reply:
Formatter adds `return if`. I considered letting the formatter recurse into branches, but then probably it should also add a `else return nothing` branch and I didn't really like that (https://github.com/fredrikekre/Runic.jl/issues/52).

After formatting a couple of code bases it is roughly 50/50 where you actually want to return things or if `return nothing` after the `if` is what is really intended.